src/DateComparison.php

```diff
@@ -17,6 +18,33 @@ public static function onSameDay(DateTimeImmutable $date1, DateTimeImmutable  $d
         return $date1->format('Y-m-d') === $date2->format('Y-m-d');
     }
 
+    public static function isThisEvening(DateTimeImmutable $date): bool
+    {
+        return self::onSameDay($date, new DateTimeImmutable()) && (int)$date->format('G') >= 18;
```

Comment:
Maybe this can benefit from Carbon https://github.com/briannesbitt/carbon?
That has a method to check this https://github.com/briannesbitt/Carbon/blob/87a5e6f250b00f4422b2859baf60fb2c5d543e97/src/Carbon/Carbon.php#L108

Reply:
But the comparisons can indeed be made simpler with carbon, but you'll still have to take care of the timezones.

Reply:
Right. But `onSameDay()` is not really in scope of this PR I think? It's an existing method that is not touched or used for these changes. It was just an example of a method where we already do the timezone conversion, which we also need in the new `isThisEvening()` etc (for which Carbon does not seem to have methods)

Reply:
Ah you already changed it. Okay 👍